applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -1,6 +1,11 @@
 import KratosMultiphysics
-import KratosMultiphysics.CompressiblePotentialFlowApplication as CompressiblePotentialFlowApplication
-#from CompressiblePotentialFlowApplication import*
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+
+def DotProduct(A,B):
```

Comment:
same comment as in the other PR

Reply:
Do you mean this comment https://github.com/KratosMultiphysics/Kratos/pull/4642#discussion_r274904021? I understand and agree that it is better to do this kind of thing in c++, however, we should find a solution until we complete migrate this process to c++. Ideas:

1. We delete this function and use numpy (the idea of @marcnunezc originally was to avoid to import numpy to reduce the dependency on external libraries as commented in other reviews)
2. We delete this function and do it by hand.

Any other idea?